code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -62,7 +64,7 @@
 	inhand_icon_state = "magic_bullet"
 	special = "This weapon fires extremely slowly. \
 		This weapon pierces all targets. \
-		This weapon fires significantly faster wearing the matching armor"
+		This weapon becomes stronger when wearing the matching armor"
```

Comment:
Put this on the Big Iron armor.

Reply:
![image](https://github.com/vlggms/lobotomy-corp13/assets/79374236/790350db-8f19-4ec2-8518-e6035a32495c)